utils/ami-cleanup/ami_cleanup.py

```diff
@@ -51,6 +51,12 @@ def get_arguments():
     return parser.parse_args()
 
 
+def parse_dry_run_value(argument):
+    if argument.lower() == "false":
+        return False
+    else:
+        return True
```

Comment:
```suggestion
    return argument.lower() == "false"
```

Reply:
* Added test for no argument supplied.
* I like your laconic style but mine is easier to read